VMD_scripts/Blob_GUI.tcl

```diff
@@ -214,11 +305,13 @@ proc ::blobulator::graphRepUser {} {
 	set count 0
 
 	set sel [atomselect $::blobulator::MolID protein]
+	puts [$sel get user2]
 	set user2length [lsort -unique [$sel get user2]]
-	$sel delete
+	puts $user2length
+	$sel delete 
 	foreach u2 $user2length {
 
-		mol representation QuickSurf 1.21 1 .5 2
+		mol representation QuickSurf 1.21 1 .5 3
```

Comment:
what's the purpose of this change?

Reply:
This increases the resolution of the quicksurf represnetations.